packages/cactus-odap-hermes/src/main/json/openapi.json

```diff
@@ -0,0 +1,1216 @@
+{
+    "openapi": "3.0.3",
+    "info": {
+        "title": "Hyperledger Cactus Plugin - Connector Besu",
```

Comment:
@jscode017 Please update the title to talk about ODAP Hermes, not the Besu Connector ;-)

Reply:
ok, solve this with new commits